patches/chromium/accessible_pane_view.patch

```diff
@@ -0,0 +1,30 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Cheng Zhao <zcbenz@gmail.com>
+Date: Thu, 4 Oct 2018 14:57:02 -0700
+Subject: fix: add back virtual methods in AccessiblePaneView
+
+Mark SetPaneFocus and RemovePaneFocus as virtual in AccessiblePaneView, as we
```

Comment:
Can you open the CL to chromium and add that CL to this patch so that we can more easily follow/track this patch to ensure it's deleted in future?

Reply:
Submitted https://crrev.com/c/1959189.